x-pack/plugins/apm/public/components/app/service_inventory/ServiceList/index.tsx

```diff
@@ -148,12 +146,9 @@ export function getServiceColumns({
       : []),
     {
       field: 'avgResponseTime',
-      name: i18n.translate(
-        'xpack.apm.servicesTable.avgResponseTimeColumnLabel',
-        {
-          defaultMessage: 'Avg. response time',
-        }
-      ),
+      name: i18n.translate('xpack.apm.servicesTable.avgLatencyColumnLabel', {
+        defaultMessage: 'Avg. latency',
```

Comment:
```suggestion
        defaultMessage: 'Latency (avg.)',
```

To align with the new column name

<img width="1205" alt="Screenshot 2021-01-13 at 11 26 37" src="https://user-images.githubusercontent.com/4104278/104440112-3c49fc80-5592-11eb-8ab1-ffcedf9eb2a0.png">